ecl2df/summary.py

```diff
@@ -427,7 +450,7 @@ def _fix_dframe_for_libecl(dframe: pd.DataFrame) -> pd.DataFrame:
             str({colname.partition(":")[0] + ":*" for colname in block_columns}),
         )
 
-    return dframe
+    return dframe.drop("DATE", axis="columns", errors="ignore")
```

Comment:
aren't any potential `DATE` column already dropped when in the line:
`dframe.set_index(dframe["DATE"], inplace=True)`? which will run if `DATE` is in `dframe.columns`?

Reply:
Only when I fix the `set_index(dframe["DATE"])` to `set_index('DATE')`. Fixed.